### PR TITLE
PYTHON-3504 Python 3.11 Support for PyMongo Auth Aws

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,13 +40,13 @@
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.4",
```
Comment (ShaneHarvey) [inline setup.py:43]: You'll need to update python_requires if we want to actually drop support for 3.4

**Review by ShaneHarvey — CHANGES_REQUESTED** [review]
Could you mention dropping 3.4 support in the changelog and also create a separate Jira ticket to track?

File: setup.py (hunk 1 above)
Comment (blink1073) [reply setup.py:43]: Done.  PyMongo 3.13 doesn't support 3.4, which is why we have to drop it.

Comment (ShaneHarvey) [reply setup.py:43]: I think you forgot to push.

Comment (blink1073) [reply setup.py:43]: Correct, done

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**